pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -129,6 +130,19 @@ func (rc *reconciler) syncStates() {
 	}
 }
 
+func (rc *reconciler) isExclusiveVolume(volumeSpec *volume.Spec) bool {
+	if volumeSpec.PersistentVolume == nil {
+		return false
+	}
+	// check if this volume can be attached to multiple PODs/nodes, if yes, return false
+	for _, accessMode := range volumeSpec.PersistentVolume.Spec.AccessModes {
```

Comment:
when attaching volumes, we use readOnly true or false to determine to attach as "ro" or "rw". This information is not coming from PersistentVolume.Spec.AccessModes